.storybook/utils/styled-system-props.js

```diff
@@ -751,7 +766,9 @@ const StyledSystemProps = ({
     rows.push(...generateStyledSystemWidthProps(defaults));
   }
   if (flexBox) {
-    rows.push(...generateStyledSystemFlexBoxProps(defaults));
+    rows.push(
+      ...generateStyledSystemFlexBoxProps(defaults, flexBox === "with gap")
```

Comment:
Another thing to think about here, which might make the above redundant: since this is our own prop and not actually part of styled system, maybe it should be in the "main" props table rather than one of the styled system ones? (Since most of those are "CSS-y" things like `overflowWrap` anyway.)

Reply:
I did consider using a `hasGap` prop but decided it wasn't as needed we just checking for the "truthyness" of it, personally think either approach is fine but chose the current as this is likely to be the only time we need to update the flexbox prop table. Wrt to where it's listed in the prop table, I prefer it in the `flexbox` subsection as it will only work when the Box is flexed etc; the other props, on the other hand` are independent.

Reply:
I've ended up amending as it was trivial to do